lib/cereal/grpc/grpc.go

```diff
@@ -80,9 +82,30 @@ func (c *workflowCompleter) EnqueueTask(task *backend.Task, opts backend.TaskEnq
 	c.tasks = append(c.tasks, t)
 	return nil
 }
+func (c *workflowCompleter) finish(err error) error {
```

Comment:
```suggestion

func (c *workflowCompleter) finish(err error) error {
```

Reply:
I looked at this for like 15 seconds trying to figure out wtf you changed. Whitespace :man_facepalming: